docs/4.x/apis.md

```diff
@@ -105,6 +105,25 @@ Any change would usually fall under one of these categories:
 
 If the change is a new config or a config change, then it's usually not mentioned in the developer changelog as they are mostly meant for users and not for developers.
 
+## PHP Plugin API
+
+### Process for declaring something as public API for plugins
+
+We provide a [public PHP API for plugins](https://developer.matomo.org/api-reference/classes) so plugins can extend and customise Matomo. For example, they allow plugins to define new reports, widgets and track additional data. They also provide utility methods to store data, to show notifications, to define settings and more.
+
+We aim to make as few classes as needed a public API yet as many as possible. Generally, to make a method or a class a public API the following criteria needs to be fulfilled:
```

Comment:
```suggestion
We aim to make as few classes as needed a public API, since they will be required to remain backwards compatible, while still providing plugin developers with a large enough stable API to effectively build with. Generally, to make a method or a class a public API the following criteria needs to be fulfilled:
```

Reply:
thought this sounded a bit clearer